test/lint/lint-includes.py

```
@@ -14,14 +14,16 @@
 
+from lint_ignore_dirs import SHARED_EXCLUDED_SUBTREES
+
 
 EXCLUDED_DIRS = ["src/leveldb/",
                  "src/crc32c/",
                  "src/secp256k1/",
                  "src/minisketch/",
                  "src/dashbls/",
                  "src/immer/",
-                 "src/crypto/x11/"]
+                 "src/crypto/x11/"] + SHARED_EXCLUDED_SUBTREES
```

Ruff (0.15.12)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)